uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-requests.js

```diff
@@ -356,6 +357,13 @@ define([
      * @return {Promise}
      */
     function loadLeaveRequest (type) {
```

Comment:
`loadLeaveRequests`, it's plural. Otherwise it looks like this method is used to load a specific leave request